guide/src/main/asciidoc/problems/resourceNotFound.adoc

```diff
@@ -41,9 +41,15 @@ Content-Type: application/problem+json
       {
         "in": "path",
         "name": "enterpriseNumber",
-        "detail": "the enterprise number 0206731645 is not assigned",
+        "detail": "The enterprise number 0206731645 is not assigned",
         "value": "0206731645"
       }
   ]
 }
 ```
+
+ifdef::full-guide[]
+Note that this problem type is only used when the resource path cannot be resolved. For resources referenced otherwise (i.e. request body, header or query parameter), <<bad-request>> is returned with a `urn:problem-type:belgif:input-validation:referencedResourceNotFound` issue type.
+
+The <<input-validation-schema,InputValidationProblem Schema Object>> SHOULD be used to represent this type of problem.
```

Comment:
I would put this paragraph before the previous one. Because currently it could be interpreted as a comment about BadRequestProblem?